packages/querygrid/src/query/api.ts

```diff
@@ -255,14 +264,15 @@ class Renderers {
 
     static applyColumnRenderer(columnMetadata, rawColumn, metadata) {
         let value = this._check(columnMetadata, rawColumn, 'columnRenderer', metadata);
+
         if (value === undefined) {
             if (rawColumn.multiValue === true) {
                 value = 'MultiValueColumnRenderer';
             }
             else if (rawColumn.name === 'harvest') {
                 value = 'MaterialLookupColumnRenderer';
             }
-            else if (rawColumn.rangeURI && rawColumn.rangeURI.indexOf('fileLink') > -1) {
+            else if (rawColumn.type === 'File') {
```

Comment:
I wonder if this was originally meant to distinguish between FIles and Attachments (both are still possible as types for the List Designer).  Do both have type "File"?

Reply:
I would think that as long as both FIle and Attachment types provide the image URL to the query response, we should be able to render them both with FileColumnRenderer. As Susan mentioned, do both have the same rawColumn.type?

Reply:
I had to correct this logic, when it's a lookup the type can be "text". I had to check type and friendlyType instead.